### PR TITLE
TECH-163: Fix cucumber tests path

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
## Description
For now, cucumber tests are localized in test/gherkin folder.
Move them to the test folder.

### CODE REVIEW

File: test/.gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
 node_modules
-yarn.lock
```
Comment (dborowiecki) [inline test/.gitignore:2]: Should yarn.lock be added to gitignore even if it's commited to repo?

**Review by dborowiecki — COMMENTED** [review]
I see that several feature files were deleted in the process, please confirm that it was intentional.

File: test/.gitignore (hunk 1 above)
Comment (KarolinaKopacz) [reply test/.gitignore:2]: I thought it should be, but if we keep `yarn.lock` in the repository then we will always be able to compare the old version with the new one, so I removed yarn.lock from the .`gitignore` file

**Comment by KarolinaKopacz** [comment]
@dborowiecki deletion of some files was intentional. I have received approval from @ingmarvali

**Pull request merged**